app/components/navigation_header/header.tsx

```diff
@@ -155,7 +153,7 @@ const Header = ({
         }
 
         const barHeight = heightOffset - ViewConstants.LARGE_HEADER_TITLE_HEIGHT;
-        const val = (scrollValue?.value ?? 0);
+        const val = (scrollValue?.value || 0);
```

Comment:
why this change?

Reply:
scrollValue is not null, so I didn't think a null coalesce is necessary:

![Screenshot 2024-07-25 at 5 21 26 PM](https://github.com/user-attachments/assets/bb723dff-9a67-4901-b54b-e693be9d279b)